pkg/object/object.go

```diff
@@ -37,3 +37,9 @@ func (o *Object) ToV2() *object.Object {
 
 	return nil
 }
+
+// HeaderJSON marshals object's header
```

Comment:
```suggestion
// MarshalHeaderJSON marshals object's header
```

Reply:
thank you, rewrite that more than one time since did not know how to name it 

btw, is naming ok?

Reply:
For me its ok